front/components/assistant/conversation/input_bar/InputBarAttachments.tsx

```diff
@@ -71,14 +71,17 @@ export function InputBarAttachments({
 
         const isWebsiteOrFolder = isWebsite(dataSource) || isFolder(dataSource);
         const visual = isWebsiteOrFolder ? (
-          <Icon visual={logo} size="sm" />
+          <Icon visual={logo} />
         ) : (
-          <>
-            <Icon visual={logo} size="sm" />
-            {getVisualForDataSourceViewContentNode(node)({
-              className: "h-5 w-5",
-            })}
-          </>
+          <DoubleIcon
```

Comment:
I feel like the spacing between icon and doc info is much closer than before

current prod:
<img width="400" alt="Screenshot 2025-04-08 at 11 32 48" src="https://github.com/user-attachments/assets/1c23cbc1-3b1e-4814-8dba-74781f881ebf" />

this PR:
<img width="381" alt="Screenshot 2025-04-08 at 11 32 20" src="https://github.com/user-attachments/assets/3c58fde6-612f-40cd-bfc5-4439bfbe3a9f" />

Reply:
Updating the DropdownItem so that the main icon is bigger